src/solver/conv_winoRxS_ultra_f2x3.cpp

```diff
@@ -0,0 +1,864 @@
+                                     params.in_width,
+                                     params.kernel_dilation_h,
+                                     params.kernel_dilation_w,
+                                     params.batch_sz,    // N
+                                     n_inputs_per_group, // K
+                                     params.out_height,
+                                     params.out_width,
+                                     params.kernel_size_h,
+                                     params.kernel_size_w,
+                                     n_outputs_per_group, // C
+                                     params);
+    }
+#else
+    (void)params; // -warning
+    return false;
+#endif
+}
+
+bool ConvBinWinogradUltraRxSf2x3::IsApplicable(const ConvolutionContext& params) const
+{
+    if(miopen::IsDisabled(MIOPEN_DEBUG_AMD_WINOGRAD_ULTRA_RXS_F2X3{}))
+        return false;
+    return IsApplicableBase(params);
+}
+
+ConvSolution
+ConvBinWinogradUltraRxSf2x3::GetSolution(const ConvolutionContext& params,
+                                         const PerformanceConfigConvBinWinogradUltraRxSf2x3& config,
+                                         const bool disableConfigOverrideFromEnv) const
+{
+    const PerformanceConfigConvBinWinogradUltraRxSf2x3* pcfg = &config;
+    PerformanceConfigConvBinWinogradUltraRxSf2x3 fromEnv;
+
+    if(!disableConfigOverrideFromEnv)
+    {
+        std::string s;
+        const auto p_asciz =
+            miopen::GetStringEnv(MIOPEN_DEBUG_AMD_WINOGRAD_ULTRA_RXS_F2X3_PERF_VALS{});
+        if(p_asciz != nullptr)
+        {
+            s = std::string(p_asciz);
+            if(!s.empty()) // else nothing to parse.
+            {
+                if(!fromEnv.Deserialize(s) || !fromEnv.IsValid(params))
+                {
+                    MIOPEN_LOG_E("MIOPEN_DEBUG_AMD_WINOGRAD_RXS_F2X3_PERF_VALS: "
+                                 "Bad format or invalid for the problem config: "
+                                 << s);
+                }
+                else
+                {
+                    MIOPEN_LOG_I("Overridden from env: " << fromEnv.ToString());
+                    pcfg = &fromEnv;
+                }
+            }
+        }
+    }
+
+    const unsigned n_groups = params.GetStream().GetMaxHardwareComputeUnits();
+    const auto group_cnt    = params.group_counts;
+    const auto intl_factor  = pcfg->GetInterleaveFactor();
+
+    constexpr unsigned F_REVERSE_R = 1 << 0;
+    constexpr unsigned F_REVERSE_S = 1 << 1;
+    constexpr unsigned F_FLIP_K_C  = 1 << 2;
+
+    int N, C, H, W, K, out_H, out_W, R, S, pad_H, pad_W, flags;
+    BuffInfo d_buf, o_buf, f_buf;
+
+    int unused               = 0;
+    uint64_t reserved_offset = 0;
+    int* reserved_ptr        = nullptr;
+    float relu_alpha         = 1.0;
+
+    if(!params.direction.IsBackwardWrW())
+    {
+        const auto is_forward = params.direction.IsForward();
+
+        GetCompiledInParameters(
```

Comment:
Do not use `GetCompiledInParameters`. Let's use `params.conv_problem` directly or `UnifiedDescriptionConv2d`.

Reply:
Thank you! Fixed in 27fd061